tasks/galera.yaml

```
@@ -0,0 +1,86 @@
+---
+- name: Check if grastate.dat exists
+  stat:
+    path: /var/lib/mysql/grastate.dat
+  register: grastate
+
+- name: Check if cluster is save to bootstrap
+  when: grastate.stat.exists
+  slurp:
+    path: /var/lib/mysql/grastate.dat
+  register: bootstrap
+
+- name: Check if mariadb is already runnning
+  stat:
+    path: /var/run/mysqld/mysqld.pid
+  register: pid
+
+- name: Fail if cluster cannot be initialzed
+  fail:
+    msg: "cluster initializer is not ready to bootstrap cluster"
+  when:
+    - mariadb.galera.initializer == True
+    - bootstrap.content is defined
+    - "'Safe to Bootstrap: 0' in bootstrap.content | b64decode"
+    - pid.stat.exists == False
+
+- block:
+
+  - name: stop mariadb
+    service:
+      name: mariadb
+      state: stopped
+
+  - name: remove wsrep_ready
+    file:
+      path: /tmp/wsrep_ready
+      state: absent
+
+  - name: repeatedly write cluster state to wsrep_ready
+    cron:
+      name: "check cluster"
+      minute: "*"
+      job: "echo \"SHOW STATUS LIKE 'wsrep_ready';\" | mysql -u root | grep 'wsrep_ready' > /tmp/wsrep_ready"
```

This could be written more readable. But see my review comment below.
```suggestion
      job: >
        echo "SHOW STATUS LIKE 'wsrep_ready';" | mysql -u root | grep 'wsrep_ready' > /tmp/wsrep_ready"
```

---

Fixed in c138ed1a122acc54fd25b0d92213ec3e5ad58704